replication_handler/components/mysql_tools.py

```diff
@@ -47,7 +47,7 @@ def restore_mysql_dump(db_creds, dump_file):
 
 def create_mysql_dump(db_creds, databases):
     temp_file = _get_dump_file()
-    dump_cmd = "mysqldump --host={} --port={} --user={} --password={} {} {} {} {} --databases {} > {}".format(
+    dump_cmd = "mysqldump --set-gtid-purged=OFF --host={} --port={} --user={} --password={} {} {} {} {} --databases {} > {}".format(
```

Comment:
i have removed this, and reverted the mysql version to default of travis which i guess is 5.6.